### ISSUE TITLE
old coach mark briefly visible when displaying unrelated coach mark sequence

### ISSUE THREAD

**Issue by glasheen99** [pseudoankit/coachmark#14]
Recreate:
1. Add this button to the demo:
```
            Button(onClick = { show(Keys.TextBottom, Keys.TextTop) }) {
                Text(text = "Highlight Some")
            }
```
2. Run the demo.
3. Tap the new button and tap through the displayed coach marks.
4. Tap "Highlight 1".

Expected behavior: Nothing is visible from the previous coach mark sequence.
Actual behavior: A "ghost" (fade-out animation?) of the *first* coach mark (highlight and tooltip) from the previous sequence is briefly visible. (at 0:05 in the video below)

[ghost1.webm](https://github.com/pseudoankit/coachmark/assets/98324119/a9941e81-8928-4415-a28f-466da0d3de7d)

Observations:
- This doesn't seem to happen (or isn't noticeable) when the two coach mark sequences start with the same coach mark.

I haven't investigated the cause.

**Comment by ankitk77** [comment]
Thanks for raising the issue, will check this

**Comment by ankitk77** [comment]
Thanks for raising the issue, It's fixed in `v1.0.5`

**Comment by glasheen99** [comment]
The ghost is gone.  Thanks for the fix!
